Ladybird/AppKit/UI/LadybirdWebViewBridge.cpp

```diff
@@ -100,6 +100,11 @@ void WebViewBridge::set_preferred_color_scheme(Web::CSS::PreferredColorScheme co
     client().async_set_preferred_color_scheme(color_scheme);
 }
 
+void WebViewBridge::mouse_wheel_event(Gfx::IntPoint position, Gfx::IntPoint screen_position, GUI::MouseButton button, KeyModifier modifiers, int wheel_delta_x, int wheel_delta_y)
+{
+    client().async_mouse_wheel(to_content_position(position), screen_position, to_underlying(button), to_underlying(button), modifiers, wheel_delta_x, wheel_delta_y);
```

Comment:
I believe there's some logic in the other chromes about the wheel delta. In the Qt one for sure, we do some sauce to this to get a more reasonable wheel increment.

Reply:
Sure that is right, I will look in to that